src/controls/listView/ListView.tsx

```diff
@@ -625,6 +625,7 @@ export class ListView extends React.Component<IListViewProps, IListViewState> {
             groupProps={groupProps}
             className={listClassName}
             onRenderDetailsHeader={this._onRenderDetailsHeader}
+            onRenderRow={onRenderRow}
```

Comment:
@milanholemans just to double-check, does this work fine when onRenderRow is not passed into the control?

Reply:
Yes, tested it in the test web part. You can omit this property and everything keeps working. 😃